README.rst

```diff
@@ -1,51 +1,103 @@
-*********
```

Comment:
I prefer we have a H1 here and then badges.

Reply:
> I prefer that we have an H1 of TrueLearn here and the badges are under it.

I dont use TrueLearn as the header, because the logo will be used and it looks much better under the badges. The picture doesnt load because I reference the image in the main branch (where it isnt yet), once we merge this you'll be able to see it.

Reply:
I believe you can use relative link here. You don't need to refer to `raw.githubusercontent.com` directly.